paper-slider.html

```diff
@@ -483,6 +488,10 @@
         'up pageup end': '_incrementKey'
       },
 
+      attached: function() {
+        this._isRTL = window.getComputedStyle(this)['direction'] === 'rtl';
```

Comment:
@frankiefu yep, I know about `:dir()` and how it doesn't work in Chrome yet.

Did you read the bug? https://github.com/PolymerElements/paper-slider/issues/190 it talks about how just checking `[dir]` is not good enough.